httpev.ml

```diff
@@ -955,6 +958,10 @@ let send_reply c cout reply =
       let push = function
       | "" -> Lwt.return_unit
       | s ->
+        let%lwt s = match c.req with
+        | Ready { encoding=Gzip; _ } -> Gzip_io.string_lwt s
```

Comment:
This doesn't seem right.
If I'm reading correctly you're Gzip-encoding each chunk. But that's not how it's meant to happen.
Compression precedes the chunking.